src/igvfd/schemas/in_vitro_system.json

```diff
@@ -590,6 +590,19 @@
                 "appscript": "DMEM with serum",
                 "igvf_utils": "DMEM with serum"
             }
+        },
+        "biosample_qualifier": {
+            "title": "Biosample qualifier",
+            "description": "Desription of cell state",
```

Comment:
Minor typo here "description". And I wonder if we should try to write additional details to explain that it is meant to be a way to capture cell states that are more precise than what can be captured by ontologies? Or if there's another better way to describe that.